src/test/java/be/seeseemelk/mockbukkit/entity/EndermanMockTest.java

```diff
@@ -0,0 +1,93 @@
+package be.seeseemelk.mockbukkit.entity;
+
+import be.seeseemelk.mockbukkit.MockBukkit;
+import be.seeseemelk.mockbukkit.ServerMock;
+import be.seeseemelk.mockbukkit.block.data.BlockDataMock;
+import org.bukkit.Material;
+import org.bukkit.block.data.BlockData;
+import org.bukkit.material.MaterialData;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.UUID;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class EndermanMockTest
+{
+
+	private ServerMock serverMock;
+	private EndermanMock endermanMock;
+
+	@BeforeEach
+	void setUp()
+	{
+		serverMock = MockBukkit.mock();
+		endermanMock = new EndermanMock(serverMock, UUID.randomUUID());
+	}
+
+	@AfterEach
+	void tearDown()
+	{
+		MockBukkit.unmock();
+	}
+
```

Comment:
Tests should be added to ensure the IllegalStateException is thrown when the enderman doesn't have a block.

Reply:
There are 2  Tests for this